jxl/src/render/stages/noise.rs

```diff
@@ -0,0 +1,357 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// TODO(zond): remove once we use this!
+#![allow(dead_code, unused_variables)]
```

Comment:
I'm not sure we need `unused_variables` here

Reply:
Removed it.